src/main/java/Duke.java

```diff
@@ -1,10 +1,208 @@
+import java.util.ArrayList;
+import java.util.Locale;
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final ArrayList<Task> LIST_OF_THINGS = new ArrayList<>();
+
+    private abstract static class Task {
```

Comment:
Maybe you could create Task as a separate class, instead of nesting it in the Duke class.